wazo_call_logd/cel_interpretor.py

```diff
@@ -639,6 +656,20 @@ def interpret_wazo_call_log_destination(self, cel, call: RawCallLog):
                 call.destination_name,
                 extra_dict['type'],
             )
+        elif extra_dict['type'] == 'queue':
+            destination_details = {
+                'type': extra_dict['type'],
+                'queue_id': extra_dict['id'],
+                'queue_label': extra_dict['label'],
+            }
+            call.destination_name = extra_dict['label']
+            call.set_tenant_uuid(extra_dict['tenant_uuid'])
```

Comment:
Why don't we do set the tenant uuid for any destination?

Reply:
:+1: I added the `tenant_uuid` to all `WAZO_CALL_LOG_DESTINATION` events